platform/android/export/export.cpp

```diff
@@ -1355,11 +1400,9 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/support_large"), true));
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/support_xlarge"), true));
 		r_options->push_back(ExportOption(PropertyInfo(Variant::BOOL, "screen/opengl_debug"), false));
-
-		for (uint64_t i = 0; i < sizeof(launcher_icons) / sizeof(launcher_icons[0]); ++i) {
-			r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, launcher_icons[i].option_id, PROPERTY_HINT_FILE, "*.png"), ""));
-		}
-
+		r_options->push_back(ExportOption(PropertyInfo(Variant::STRING, launcher_icons[0].option_id, PROPERTY_HINT_FILE, "*.png"), ""));
```

Comment:
Since you no longer need to iterate over different properties names, I guess you could just put the string here directly and drop the `option_id` member from the struct?

Reply:
Done.